### PR TITLE
change parameters to be closer to GPML, remove the kronecker delta stuff

### PR DESCRIPTION
**Pull request by drezap**
#### Submission Checklist

- [x] Builds locally 
- [x] Declare copyright holder and open-source license: see below

#### Summary
This contains the following changes, which I'd like to carry out through the rest of the chapter.

1. Change parameters to the same as in GPML.
2. Remove the kronecker delta stuff. This isn't in BDA3, or GPML. Sure, I know what a kronecker delta is, but what are we testing exactly? 

"this test is between indeces i,j..."? Introducing too many parameters too quickly makes things too difficult to understand. If it's really necessary, we can include this later in the chapter. It's too wordy and needs to be parsimonious.

I'm using a squared l-2 norm here, because they're defined the same in pretty much all of mathematics and statistics. I can explain how the covariance function corresponds to element i,j later.

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company): Andre Zapico 2020

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)

### CODE REVIEW

**Comment by drezap** [comment]
@betanalpha Sorry to tag you! I can tell you wrote it.

@avehtari, @rtrangucci 

If no one responds, I'll assume the changes are cool, and I'll just merge. Everything I remove I'll make a note to include somewhere else in chapter.

**Comment by jgabry** [comment]
Thanks @drezap. Can one of the people tagged in @drezap's previous comment give this a quick look?

**Comment by drezap** [comment]
Didn't mean to seem angry insulting, or angry :) I'll do my best to be non-hostile, and abide by the code of conduct :)

**Comment by jgabry** [comment]
> Didn't mean to seem angry insulting, or angry :) I'll do my best to be non-hostile, and abide by the code of conduct :)

Thanks, all good :)

**Comment by jgabry** [comment]
Also, I definitely support the idea of using notation in our docs that's consistent with the other resources people are most familiar with.

**Comment by drezap** [comment]
Snap, I forgot what a kronecker delta is. It's not needed.

If distance is 0, for i=j the distance becomes zero and the covariance matrix at that point is evaluated to 1. This is intentional. The whole kronecker delta thing is redundant and confusing.

I'm taking it out.

**Comment by drezap** [comment]
I'm just wasting peoples time, I'll just take this out
